region/brinstar/blue.json

```diff
@@ -1007,6 +1112,66 @@
               ],
               "openEnd": 0
             }
+          ],
+          "leaveWithGModeSetup": [
+            {
+              "strats": [
+                {
+                  "name": "Get Hit By Geemer - Break the Crumble Blocks",
+                  "notable": false,
+                  "requires": [
+                    "f_ZebesAwake",
+                    {"or": [
+                      "h_canDestroyBombWalls",
+                      {"obstaclesCleared": ["A"]}
+                    ]},
+                    {"or": [
+                      "canConsecutiveWalljump",
+                      "h_canFly"
+                    ]}
+                  ],
+                  "note": "Falling down the shaft and breaking the crumble block does not require Morph."
+                },
+                {
+                  "name": "Get Hit By Geemer - Break the Power Bomb Blocks",
+                  "notable": false,
+                  "requires": [
+                    "f_ZebesAwake",
+                    {"or": [
+                      "h_canDestroyBombWalls",
+                      {"obstaclesCleared": ["A"]}
+                    ]},
+                    "h_canUsePowerBombs",
+                    {"or": [
+                      {"canShineCharge": {
+                        "usedTiles": 33,
+                        "shinesparkFrames": 43,
+                        "openEnd": 2
+                      }},
+                      {"and": [
+                        "canShinechargeMovementComplex",
+                        "canFastWalljumpClimb",
```

Comment:
This would consecutive walljump up instead of spark... A will be destroyed by the spark, so the check for `obstacleCleared` can be removed here.